index.html

```diff
@@ -6,22 +6,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Readex+Pro:wght@500&family=Roboto:wght@900&display=swap"
+        rel="stylesheet">
     <title>Devinombre</title>
 </head>
 
 <body>
     <header>
         <h1>Devine!</h1>
-        <p>(Entre 1 et 20)</p>
-        <button>Rejouer</button>
-        <div>?</div>
+        <p class="fourchette">(Entre 1 et 20)</p>
```

Comment:
I think the class name should be in English even if the content of the page is in French.

Reply:
done in 66265116ee4680d51b8eaa5853ae7de9c6e90d72